.changeset/om-buffer-claim-ownership.md

```
@@ -0,0 +1,17 @@
+'@mastra/core': patch
+'@mastra/memory': patch
+'@mastra/server': patch
+'@mastra/libsql': patch
+'@mastra/pg': patch
+'@mastra/mysql': patch
+'@mastra/mongodb': patch
+'@mastra/convex': patch
+'@mastra/oracledb': patch
+---
+
+Fixed Observational Memory buffering so independent Mastra processes sharing one storage backend can no longer clear or overwrite each other's active buffering work. Each observation-buffer cycle now acquires an atomic storage-owned claim (a unique cycle token with a bounded lease that is renewed while the model call runs), and buffered output, lease renewal, and release are all owner-conditioned: a crashed owner's claim expires and can be taken over atomically, and a late completion from a replaced owner is fenced out instead of overwriting the successor's state. Status rendering now reports `running` only for a valid unexpired claim (or a bounded legacy marker), never for a bare boolean left behind by a dead process.
```

markdownlint-cli2 (0.23.2)

[warning] First line in a file should be a top-level heading

(MD041, first-line-heading, first-line-h1)